modules/docs/arrow-docs/docs/docs/typeclasses/foldable/README.md

```diff
@@ -9,8 +9,518 @@ permalink: /docs/typeclasses/foldable/
 {:.intermediate}
 intermediate
 
-TODO. Meanwhile you can find a short description in the [intro to typeclasses]({{ '/docs/typeclasses/intro/' | relative_url }}).
+The Typeclass `Foldable` provide us the ability of, given a type `F<A>`, fold their values `<A>`.
 
+`Foldable<F>` is implemented in terms of two basic methods:
+
+- `foldLeft(fa, b)(f)` eagerly folds `fa` from left-to-right.
```

Comment:
Automatic currying isn't possible in Kotlin. Also, we have these defined as extention functions, so they have to be changed to `fa.foldLeft(init, f)`.